src/lib/operators/insert.cpp

```diff
@@ -240,6 +247,7 @@ void Insert::_on_rollback_records() {
     for (auto chunk_offset = target_chunk_range.begin_chunk_offset; chunk_offset < target_chunk_range.end_chunk_offset;
          ++chunk_offset) {
       mvcc_data->end_cids[chunk_offset] = 0u;
+      target_chunk->increase_invalid_row_count(1u);
```

Comment:
Is there a reason why we didn't do that before?

Reply:
Probably because we forgot it :D